WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacyControlViewModel.cs

```diff
@@ -65,7 +65,9 @@ public PrivacyControlViewModel(Wallet wallet, TransactionInfo transactionInfo, T
 				.WhenValueChanged(x => x.IsSelected)
 				.Subscribe(_ =>
 				{
-					var selectedPocketLabels = Pockets.Where(x => x.IsSelected).Select(x => x.Labels);
+					var selectedPocketLabels = Pockets.Where(x => x.IsSelected)
```

Comment:
Something is wrong here. SmartCoins + user input should be used as the source of the final label, not Pockets. The result of privacy control should be a list of coins and from those, the final label can be constructed.

Reply:
fixed